Winleafs.Wpf/Views/MainWindows/MainWindow.xaml.cs

```diff
@@ -295,6 +297,8 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
         {
             var hwndSource = HwndSource.FromHwnd(new WindowInteropHelper(this).Handle);
             hwndSource.AddHook(new HwndSourceHook(WndProc));
+
+            ToastHelper.ShowNotification(string.Format(MainWindows.Resources.UsingWinleafsVersion, UserSettings.APPLICATIONVERSION), ToastLogLevel.Information);
```

Comment:
Seems to me this is an odd place to show this notification, because this is only fired when you double click on the desktop icon only when winleafs is already active. It is more logical to add this after the main window has been initialized.

Reply:
On another note, how useful is this notification? Since we are planning to add the version as a label to the main window

Reply:
I agree that this notification isn't useful but yet a simple showcase of how to make a notification

Reply:
Better point to insert the notification: OverrideUserControl, error during overriding.

Reply:
Set the notification there